### PR TITLE
Add Docker setup guide - Closes #1163

### LINKED ISSUE #1163: Create Guide: Add docker setup to Lisk Core 3.0.0
Create a new guides that explains step by step how to setup Lisk Core using docker container.

Include standard steps similar to other distributions: https://lisk.com/documentation/lisk-core/setup/binary.html

- Include all possible flags to run Lisk Core, following is an example for your reference

```
docker run -d --volume /data:/home/lisk/.lisk \
    --publish 7000:7000 --publish 8080:8080 --publish 7008:7008\ 
    --name lisk-core lisk/core:3.0.0 start --network=testnet --port 7000\ 
    --api-ws --enable-http-api-plugin --http-api-plugin-port 7008\
    --http-api-plugin-host host.docker.internal --http-api-plugin-whitelist 0.0.0.0/0 \
    --api-ws --api-ws-port 8080 --api-ws-host 0.0.0.0
```

### Additional information

https://github.com/LiskHQ/lisk-core/tree/development/docker

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1163

### CODE REVIEW

File: modules/ROOT/pages/setup/docker.adoc [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+:url_docker_install_mac: https://docs.docker.com/docker-for-mac/install/
+:url_docker_install_windows: https://docs.docker.com/docker-for-windows/install/
+:url_docker_linux_post_install: https://docs.docker.com/install/linux/linux-postinstall
+:url_xcode: https://developer.apple.com/xcode/features/
+// Project URLs
+:url_config: management/configuration.adoc
+:url_upgrade_source: update/source.adoc
+
+This document covers how to prepare a system to run Lisk Core as a Docker image-based container.
+Lisk Core version 3 does not have any external dependencies and thus does not require using `docker-compose`.
```
Comment (chris529) [inline modules/ROOT/pages/setup/docker.adoc]:
```suggestion
Lisk Core version 3 does not have any external dependencies and hence does not require using `docker-compose`.
```

File: modules/ROOT/pages/setup/docker.adoc [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+Add the `lisk` user to the `sudo` user group:
+
+[source,bash]
+----
+sudo usermod -aG sudo lisk
+----
+
+=== Install Docker
+To run Lisk in Docker a user must first install the Docker engine.
```
Comment (chris529) [inline modules/ROOT/pages/setup/docker.adoc]:
```suggestion
To run Lisk in Docker it is necessary to first install the Docker engine.
```

File: modules/ROOT/pages/setup/docker.adoc [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+----
+docker run --volume lisk-data:/home/lisk/.lisk \
+           --publish 5001:5001 \
+           --env LISK_CONSOLE_LOG_LEVEL=debug \
+           --name lisk-core \
+           lisk/core:3.0.2 \
+             start --network=testnet
+----
+
+See the xref:{url_config}[] guide for a reference of more configuration options.
```
Comment (chris529) [inline modules/ROOT/pages/setup/docker.adoc:159]:
```suggestion
See the xref:{url_config}[] guide which is a reference that contains more configuration options.
```

File: modules/ROOT/pages/setup/docker.adoc [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+
+Example with all possible flags set:
```
Comment (chris529) [inline modules/ROOT/pages/setup/docker.adoc]:
```suggestion
An example with all possible flags set is shown below:
```

File: modules/ROOT/pages/setup/docker.adoc [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+:url_docker_install_linux: https://docs.docker.com/engine/installation/#server
+:url_docker_install_mac: https://docs.docker.com/docker-for-mac/install/
+:url_docker_install_windows: https://docs.docker.com/docker-for-windows/install/
+:url_docker_linux_post_install: https://docs.docker.com/install/linux/linux-postinstall
+:url_xcode: https://developer.apple.com/xcode/features/
+// Project URLs
+:url_config: management/configuration.adoc
+:url_upgrade_source: update/source.adoc
+
+This document covers how to prepare a system to run Lisk Core as a Docker image-based container.
```
Comment (nagdahimanshu) [inline modules/ROOT/pages/setup/docker.adoc]:
```suggestion
This document describes how to build a system to run Lisk Core as a Docker image-based container.
```
or
```suggestion
This document describes how to run Lisk Core as a Docker image-based container.
```

File: modules/ROOT/pages/setup/docker.adoc (hunk 2 above)
Comment (nagdahimanshu) [inline modules/ROOT/pages/setup/docker.adoc]:
```suggestion
To run Lisk Core in Docker, a user must first install the Docker engine.
```

**Review by chris529 — COMMENTED** [review]
Looks good, just a few minor points 👍

File: modules/ROOT/pages/setup/docker.adoc (hunk 2 above)
Comment (Tschakki) [reply modules/ROOT/pages/setup/docker.adoc]:
```suggestion
To run Lisk Core in Docker, it is necessary to first install the Docker engine.
```

**Review by nagdahimanshu — APPROVED** [review]
LGTM 👍

**Pull request merged**